domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/PartyUtils.java

```diff
@@ -25,62 +25,62 @@ private PartyUtils() {
         // utility class, no instances
     }
 
-    public static String getType(final Party claimant) {
-        if (claimant instanceof Individual) {
+    public static String getType(final Party party) {
```

Comment:
this party name change and the test change should be pulled out to a separate PR

Reply:
please?

Reply:
@geek-soft Thoughts?

Reply:
I will do it to speed things up.